### DISCUSSION TITLE
VEP Version

### DISCUSSION THREAD

**Discussion by KaitlynSparks** [broadinstitute/seqr#2886] [Q&A]
Which VEP version does seqr use? Our group is having an issue where some of our variants are not being labelled with the gene name or predicted consequence despite coming up when using the online Ensembl VEP tool. Could it have something to do with the VEP version being used? Thank you in advance.

**Comment by hanars** [comment]
seqr uses VEP 99

**Reply by KaitlynSparks** [reply]
Thank you for your rapid response. Could this possibly be causing our issue, or should I continue looking for problems our side? Would it be more helpful for me to create an issue or feature request related to this?

**Reply by hanars** [reply]
I suppose it could be the problem, although our group has been using this version of VEP for years with no problems at all. You could try installing a different version of VEP in the pipeline runner image and see if that fixes the issue if you wanted to see if that was the case. 

Please keep all discussion of the issue here for now, a feature request or an issue would be helpful if there were a specific code change you needed our team to make but that does not appear to be the case for now.
